### ISSUE TITLE
Stop repeating the color request code

### ISSUE THREAD

**Issue by tuantr10** [devinmatte/wasp#24]
For now the `theme_color` and `background_color` function have almost the same code, we can refactor and put those code into one global function. Something like this:
```
def request_color(color_type):
    ...#repeated code
    return ...

class Manifest:
    def theme_color(self):
        self.details['theme_color'] = request_color('theme_color')

    def background_color(self):
        self.details['background_color'] = request_color('background_color')
```
What do you think @devinmatte?

**Comment by devinmatte** [comment]
@tuantr10 This would likely be a good idea. However I'm going to put it on hold until after #21.

**Comment by tuantr10** [comment]
Thanks for your feedback. Lets wait until then.

**Comment by devinmatte** [comment]
@tuantr10 #21 Has been closed. You can feel free to update this if you wish. You'll notice it's not all that much different after the refactor

**Comment by davidmackhello** [comment]
@devinmatte it looks like this update was originally suggested by @tuantr10 but is now set to first-timers-only. I would like to take a crack at this if eligible... I'm a beginner teaching myself Python, and this seems doable. Let me know if I can go ahead with it.

**Comment by devinmatte** [comment]
@machajew Feel free to go ahead and give it a go. Submit a pull request whenever you get it done and I'll review it.
